src/ucp/api/ucp.h

```diff
@@ -850,6 +862,30 @@ typedef struct ucp_worker_params {
 } ucp_worker_params_t;
 
 
+/**
+ * @ingroup UCP_WORKER
+ * @brief UCP listener attributes.
+ *
+ * The structure defines the attributes which characterize
+ * the particular listener.
+ */
+typedef struct ucp_listener_attr {
+    /**
+     * Mask of valid fields in this structure, using bits from
+     * @ref ucp_listener_attr_field.
+     * Fields not specified in this mask will be ignored.
+     * Provides ABI compatibility with respect to adding new fields.
+     */
+    uint64_t              field_mask;
+
+    /**
+     * The port on which the listener is listening for incoming connection
+     * requests.
+     */
+    int                   port;
```

Comment:
Shell we explicitly specify that the port is returned and host format ?

Reply:
@shamisp done.